### ISSUE TITLE
GroupInput is wrong type.

### ISSUE THREAD

**Issue by darksnow** [uselagoon/machinery#36]
I was trying to use `AddGroupToProject` and I think I've found a discrepancy.

type `ProjectGroupsInput{}` takes a `ProjectInput{}` which takes a `uint` `Id` along with the project name.

It also gets an array of `GroupInput`. which is an alias for `ProjectInput` so takes the same `Id` and `Name`.

However a group id is not a `uint`, it's a `uuid.Uuid` in type `Group{}`

So, my question is, do we need the Name at all if we have the Id for both `ProjectInput` and `GroupInput`? More importantly, should we have a new type:

     type GroupInput struct {
       Id *uuid.Uuid,
       Name string,
     }

**Comment by shreddedbacon** [comment]
Yeah, seems like it is a discrepancy. But you can use it now if you use the `name` only when interacting with groups. You don't need to provide both fields in the input

**Comment by darksnow** [comment]
Thanks for that. I was able to use `AddGroupToProject()` using the group name without the ID.
